api/sign.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var sjcl = require('ripple-lib').sjcl;
+var Seed = require('ripple-lib').Seed;
+var SerializedObject = require('ripple-lib').SerializedObject;
+var validate = require('./lib/validate');
+
+var HASH_TX_ID = 0x54584E00; // 'TXN'
```

Comment:
Then let's please add some commentary over what the magic numbers mean, or point to the rippled documentation